products/idn/docs/identity-now/event-triggers/early-access/identity-deleted.md

```diff
@@ -20,7 +20,21 @@ This is an early access event trigger. Please contact support to have it enabled
 
 ![Flow](./img/identity-deleted-path.png)
 
-Identity deleted events occur when an identity's associated account is deleted from the identity's authoritative source. After accounts are aggregated and the identity refresh process finds an identity that is not correlated to an account, the associated identity is deleted from IdentityNow. For more information, see [Configuring Correlation](https://community.sailpoint.com/t5/Connectors/Configuring-Correlation/ta-p/74045). The Identity deleted event contains any identity attributes as they are configured in the identity profile. For more information, see [Mapping Identity Profiles](https://community.sailpoint.com/t5/Admin-Help/Mapping-Identity-Profiles/ta-p/77877).
+Identity deleted event will occur when an identity meets all of the following requirements:
+
+- No correlated accounts
+- Not an owner of a role, access profile, application, source, or taskResult
+- Not an owner or requester of a workItem
+- Not a protected account or manager
+- No assigned capabilities
```

Comment:
There is on more requirement - that the identity is not involved in any active certification as a target (its access is not being certified).

Reply:
Thanks @piotr-wojcik-sp .  I updated this.  Can you please review again?